diagnostic.md

```diff
@@ -35,19 +34,18 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
-
+aboslute
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
+absolute. The internet file is hosted in the same file path structure if it was hosted locally
```

Comment:
It's not clear what you mean here, but a file on the internet is accessed through the URL, which is its path. You can tell that it is absolute because that file will always be access using the same exact path, no matter where you are coming from.